tests/test_transpilers_placer.py

```diff
@@ -158,19 +158,19 @@ def test_random(reps):
 
 
 @pytest.mark.parametrize("gates", [None, 5, 13])
-def test_reverse_traversal_technique(gates):
+def test_revers_traversal(gates):
```

Comment:
```suggestion
def test_reverse_traversal(gates):
```

Reply:
I think it was fixed in the last commit `1466583`.